api/handlers/exporter.go

```diff
@@ -79,7 +81,17 @@ func (e *Exporter) Decideds(w http.ResponseWriter, r *http.Request) error {
 			from := phase0.Slot(request.From)
 			to := phase0.Slot(request.To)
 
-			participantsList, err := qbftStore.GetParticipantsInRange(msgID, from, to)
+			share, ok := e.Shares.Get(nil, pubKey)
+			if !ok {
+				return api.Error(fmt.Errorf("share not found"))
+			}
+
+			operatorIDs := make([]spectypes.OperatorID, 0, len(share.Committee))
+			for _, sm := range share.Committee {
+				operatorIDs = append(operatorIDs, sm.Signer)
+			}
```

Comment:
this would be wrong if the validator has moved between committees or missing if it was removed

Reply:
there's probably a similar problem today when searching by operator or committee, however AFAIK everyone is still searching by pubkey/index like before the fork